### PR TITLE
fix(snapshot): remove clone parameters

### PR DESCRIPTION
**Pull request by hrudaya21**
Clone related gRPC message removed from 2.3 version.

### CODE REVIEW

File: io-engine/src/grpc/v1/snapshot.rs [hunk 1]
```diff
@@ -599,133 +599,17 @@ impl SnapshotRpc for SnapshotService {
...
-                    }
-                })?;
-                rx.await
-                    .map_err(|_| Status::cancelled("cancelled"))?
-                    .map_err(Status::from)
-                    .map(Response::new)
-            },
-        )
-        .await
+        todo!("Not implemented");
```
Comment (tiagolobocastro) [inline io-engine/src/grpc/v1/snapshot.rs]: not this todo, this will panic!
Have to result GrpcResult with status unimplemented

Comment (tiagolobocastro) [reply io-engine/src/grpc/v1/snapshot.rs]: Might be better to remove the service endpoints the mayastor-api protobuf completely, then we just remove these completely.

Comment (hrudaya21) [reply io-engine/src/grpc/v1/snapshot.rs]: Right. Removed clone related gRPC request.

File: io-engine/src/grpc/v1/replica.rs [hunk 2]
```diff
@@ -84,7 +84,7 @@ impl From<LvolSpaceUsage> for ReplicaSpaceUsage {
 impl From<Lvol> for Replica {
     fn from(l: Lvol) -> Self {
         let usage = l.usage();
-        let source_uuid =
+        let _source_uuid =
```
Comment (tiagolobocastro) [inline io-engine/src/grpc/v1/replica.rs]: Can we completely remove this line?

Comment (hrudaya21) [reply io-engine/src/grpc/v1/replica.rs]: fixed

**Comment by hrudaya21** [comment]
bors try

**Review by tiagolobocastro — APPROVED** [review]

**Comment by hrudaya21** [comment]
bors try

**Comment by hrudaya21** [comment]
bors try

**Comment by tiagolobocastro** [comment]
bors try

**Comment by hrudaya21** [comment]
bors try

**Review by mtzaurus — APPROVED** [review]

**Comment by hrudaya21** [comment]
bors merge

**Comment by tiagolobocastro** [comment]
bors merge

**Pull request merged**